_categories/installing_gazebo.md

```diff
@@ -123,15 +118,15 @@ subcategories:
         desc: 'Instructions to build gazebo in Windows7 using MSVC12'
         star: true
 
-      - title: 'MSVC versions different than MSVC12'
-        url: 'http://answers.gazebosim.org/question/TODO' # todo
-        desc: 'Status of the support for different versions of Microsoft Visual Studio C++ compiler'
-        star: false
+      # - title: 'MSVC versions different than MSVC12'
+      #   url: 'http://answers.gazebosim.org/question/TODO' # todo
+      #   desc: 'Status of the support for different versions of Microsoft Visual Studio C++ compiler'
+      #   star: false
 
-      - title: 'Windows10 or other Windows versions installation'
-        url: 'http://answers.gazebosim.org/question/TODO' # todo
-        desc: 'Status of the support for different versions of Microsoft Windows, particularly Windows 10.'
-        star: false
+      # - title: 'Windows10 or other Windows versions installation'
```

Comment:
The current support for Windows in gazebo9 is somehow operational at this moment but I need to update the docuementation. We might want to refer to this issue by now https://bitbucket.org/osrf/gazebo_tutorials/issues/92/remove-or-update-windows-installation